app/code/Magento/InventoryInStorePickup/Model/PickupLocation/Mapper.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model\PickupLocation;
+
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\InventoryInStorePickup\Model\PickupLocation\Mapper\CreateFromSource;
+use Magento\InventoryInStorePickupApi\Api\Data\PickupLocationInterface;
+
+class Mapper
```

Comment:
Please describe the purpose of the class

Reply:
Added.